lib/chef/knife/ssh.rb

```diff
@@ -98,6 +98,10 @@ class Ssh < Knife
         :boolean => true,
         :default => true
 
+      option :sudo_password,
+        :long => "--sudo-password PASSWORD",
+        :description => "The password to use for 'sudo' commands. WARNING: using this option has limited security issues"
```

Comment:
Without mentioning what the "limited security issues" are, I don't think that this warning in the help output is very helpful.  What do you see as the security issues with this option?